String shape
String tool # used for marker
geometry_msgs/Point start_location #in top left aruco frame